addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -116,8 +116,8 @@ const Link = Widget.extend({
             'text-center', 'text-right', 'text-justify', 'text-truncate',
```

Comment:
@robinlej I think this PR should be split into two no ?
The first part about `CTA` should target 15.0 but the other part should only target saas-16.5 as the upgrade script is targeting this version no ?

Reply:
I just want those 3 lines to be done, but... you're right. :grimacing: 
Here's the other PR (saas-16.5 being 17.0): https://github.com/odoo/odoo/pull/151291